dracut.sh

```diff
@@ -1295,6 +1296,13 @@ else
     exit 1
 fi
 
+if [[ $persistent_policy == "mapper" ]]; then
+    unset persistent_policy
+elif [[ -n $persistent_policy && ! -d "/dev/disk/${persistent_policy}" ]]; then
```

Comment:
Regarding the use of "-d" in the test - are these always directories or can they be e.g. symlinks to a directory ?

Reply:
AFAIK directories.